README.md

```diff
@@ -136,6 +137,67 @@ Body (json)
 -   `createdAt: string` - an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date indicating the creation time
 -   `updatedAt: string` - an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date indicating the last updated time
 
+### PUT /miniurl/url/:id
+
+Update the URL and retrieve its information
+
+#### Request
+
+Body (json)
+
+-   `url: string` (required) - The updated url
+
+Query
+
+-   `id: string` (required) - The saved shorten url's id
+
+#### Response
+
+Body (json)
+
+-   `id: string` - The url's id
+-   `url: string` - The **updated** url
+-   `createdAt: string` - an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date indicating the creation time
+-   `updatedAt: string` - an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date indicating the last updated time
```

Comment:
Some things are missing here. For example the URL info and the bearer token for the delete/update operations.

Reply:
do we want to return extended info data for PUT operation ? 
I do added a comment for Del "`**Required:** auth bearer token."
`

Reply:
It's not the extended info.

You can't update or delete a URL without being authorized, obviously.
@Danielg212